policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/solidarity/eligibility/income/married.yaml

```diff
@@ -0,0 +1,10 @@
+description: Quebec solidarity tax credits basic family income for individual with a spouse.
```

Comment:
needs a verb. is this an income limit? phase-out start?

Reply:
income limit